spec/scenic/statements_spec.rb

```diff
@@ -108,17 +108,6 @@ module Scenic
           .with(:name, definition.to_sql)
       end
 
-      it "fails to replace the materialized view in the database" do
```

Comment:
@calebthompson At the time I tried to and was running into a few errors that I didn't quite understand and didn't have the time to fix. If this is a blocker for getting this merged I can try again.